### PR TITLE
Add Forklift to the umbrella

### PR DESCRIPTION
**Pull request by jdenen**
- Merges forklift into the umbrella
- Configures forklift for being a part of the umbrella
- Update andi dependencies as needed
- Adds `mix test` alias (temporary pending `brook` updates)

### CODE REVIEW

**Comment by jdenen** [comment]
I made changes in the last 4 commits, so you'll probably want to focus on those. Feel free to re-review the entire history of Forklift if you'd like, though.

**Comment by jdenen** [comment]
Yes. That change was made before this PR, but there is some stuff there that needs to be tweaked. I'll probably do that as part of my next PR (EOF messages) or a third for that specifically.

File: apps/forklift/rel/plugins/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.*
```
Comment (jeffgrunewald) [inline apps/forklift/rel/plugins/.gitignore]: this still needed?

File: apps/forklift/rel/config.exs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+~w(rel plugins *.exs)
```
Comment (jeffgrunewald) [inline apps/forklift/rel/config.exs]: this needed?

File: apps/forklift/rel/prod.vm.args [hunk 3]
```diff
@@ -0,0 +1,7 @@
+-name <%= release_name %>@${ERLANG_NAME}
```
Comment (jeffgrunewald) [inline apps/forklift/rel/prod.vm.args]: this needed?

File: apps/forklift/rel/vm.args [hunk 4]
```diff
@@ -0,0 +1,7 @@
+-name <%= release_name %>@127.0.0.1
```
Comment (jeffgrunewald) [inline apps/forklift/rel/vm.args]: this needed?

**Pull request merged**